README.md

```diff
@@ -1,8 +1,14 @@
 # Knowngithub
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/knowngithub`. To experiment with that code, run `bin/console` for an interactive prompt.
+> ### _"It's not paranoia if they're out to get you."_
+>
+> _- Some Wise Man_
 
-TODO: Delete this and the text above, and describe your gem
+Have you hit the problem where you're trying bootstrapping new containers or VMs and git will refuse to clone stuff because it doesn't trust GitHub? This gem answers the scenario where you believe that manually burning the known hosts into whatever you're making doesn't scale, and blindly trusting any host to provide you with legitimiate code is insufficient.
```

Comment:
The truly paranoid pin the GitHub SSL cert/intermediate cert to make sure they aren't being served random data from a malicious server.